### DISCUSSION TITLE
getAllAssets has been removed from the typescript API

### DISCUSSION THREAD

**Discussion by Moltenkore** [immich-app/immich#10356] [Q&A] [answered]
### The bug

I'm using the typescript lib and It seems getAllAssets was removed in this PR https://github.com/immich-app/immich/pull/9715 meaning my script no longer works!

Is there a similar endpoint I can use to run through all uploaded assets?

### The OS that Immich Server is running on

n/a

### Version of Immich Server

v1.106.4

### Version of Immich Mobile App

n/a

### Platform with the issue

- [ ] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

```bash
n/a
```

**Comment by mmomjian** [comment, marked as answer]
You can use the following endpoint with no params to page through all assets. As such, getallassets was duplicated and not needed: https://immich.app/docs/api/search-metadata
